design.md

```diff
@@ -1,13 +1,25 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+Given the complexity of the problem, the user will be expected to be familiar with how picobot works. So, they will already need to know that picobot has 100 possible states, that it can only check if there are walls to the north / south / east / west. They syntax also looks kind of like python's, so people who are familiar with python might have a slight edge (it was not my original idea to steal Python's syntax, but I found it helped make my code a lot more compact and was one of the last changes I made to my desigh).
 
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
+I think there are two major changes that really help make writing programs easier in my design. First, states are named, not numbered. This helps the programmer keep track of what that state is supposed to accomplish or what they know is true in that state. Second, NEWS is replaced with up / right / down / left, which is more natural for the problem.
+
```

Comment:
We made (or tried to make) the same changes. I agree, both help immensely with readability and debugging!